openlibrary/catalog/add_book/__init__.py

```diff
@@ -303,6 +303,19 @@ def update_ia_metadata_for_ol_edition(edition_id):
     return data
 
 
+def normalise_isbns(rec):
```

Comment:
This looks like a duplicate of existing functionality we have in https://github.com/internetarchive/openlibrary/blob/master/openlibrary/utils/isbn.py -- can we use this instead?

Reply:
Good catch, although I'm not sure I want to discard the data if `isbnlib.canonical()` returns `None` in this case. I'll think on it, and check what conditions `canonical()` does return none. It might prompt me to modify the utils/isbn.py method to be more tolerant of invalid check digits that might be legitimately be printed on books.

Reply:
> It might prompt me to modify the utils/isbn.py method to be more tolerant of invalid check digits that might be legitimately be printed on books.

Or at very least, let's keep this isbn logic in one file and import it, however we go about it.

Reply:
canonical doesn't do any check digit processing or anything else fancy. It just strips everything except digits and X/x

Reply:
Thanks @tfmorris, you are correct, and that's what I want here. I'll update the comments on /utils/isbn.py to remove the 'ensure length' comment which incorrectly implies we are doing more manipulations.

`canonical("test56") => "56"`
`canonical("test") = > ""`
and ours
`normalize_isbn("test56") => "56"`
`normalize_isbn("test") => None`